ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -309,6 +314,18 @@ def test_function(
         target_fn, *args, test_compile=test_flags.test_compile, **kwargs
     )
 
+    if test_flags.with_copy:
+        array_fn = ivy.is_array
```

Comment:
We shouldn't use global Ivy here.

Reply:
Alright. I'll have it changed to `ivy_backend`